charts/datadog/ci/kubeval.yaml

```diff
@@ -23,6 +23,11 @@ datadog:
   clusterChecks:
     enabled: true
   kubeStateMetricsEnabled: true
+  securityAgent:
+    compliance:
+      enabled: true
```

Comment:
When I try to enable the compliance agent with only those settings and this PR, my agents pods remain stuck with the following issue:
```
  Warning  FailedMount  2m10s               kubelet, gke-lenaic-huard-ubuntu-70baef2d-rn3a  Unable to attach or mount volumes: unmounted volumes=[complianceconfigdir], unattached volumes=[config datadog-agent-docker-token-9f786 logpodpath logdockercontainerpath runtimesocketdir installinfo debugfs complianceconfigdir sysprobe-socket-dir seccomp-root src procdir sysprobe-config pointerdir passwd datadog-agent-security modules cgroups group hostroot]: timed out waiting for the condition
  Warning  FailedMount  80s (x16 over 17m)  kubelet, gke-lenaic-huard-ubuntu-70baef2d-rn3a  MountVolume.SetUp failed for volume "complianceconfigdir" : configmap "datadog-compliance-benchmarks" not found
```

It looks like, by default, the chart tries to mount a ConfigMap that it doesn’t create.

Reply:
Ah, so actually the opposite should happen - we shouldn't mount anything by default (right now this will result in an empty dir). This is coming with https://github.com/DataDog/datadog-agent/pull/6155.